pycat.py

```diff
@@ -228,7 +231,10 @@ def main():
     port = int(sys.argv[2])
     arg = sys.argv[3] if len(sys.argv) == 4 else None
     ses = Session(world_module, port, arg)
-    ses.run()
+    try:
+        ses.run()
+    except KeyboardInterrupt:
+        raise
```

Comment:
What do these 2 lines do?

Reply:
It catches the keyboard interrupt to mute a crash in run, then passes it up to main. Without the raise, the user has to hit C^c twice